cmd/kyma/alpha/create/module/module.go

```diff
@@ -0,0 +1,137 @@
+package module
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/pkg/module"
+	"github.com/mandelsoft/vfs/pkg/osfs"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *Options
+	cli.Command
+}
+
+//NewCmd creates a new kyma CLI command
+func NewCmd(o *Options) *cobra.Command {
+
+	c := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "module MODULE_NAME MODULE_VERSION <CONTENT_PATH> [flags]",
+		Short: "Creates a module bundled as an OCI image from the contents of the given path",
+		Long: `Use this command to create a Kyma module.
+
+### Detailed description
+
+Kyma modules are individual components that can be deplyed into a Kyma runtime. Modules are built and distributed as OCI continer images. 
+This command provides the means to create such images out of a folder's contents.
```

Comment:
Trying to phrase it simpler and shorter.

```suggestion
With this command, you can create such images out of a folder's contents.
```